.github/workflows/awstest.yml

```diff
@@ -6,11 +6,13 @@ on:
   push:
     branches:
       - master
+      - dev
```

Comment:
```suggestion
```

Reply:
Will be fixed in https://github.com/nf-core/viralrecon/pull/118